### DISCUSSION TITLE
How to mute tracks when there are multiple audio tracks

### DISCUSSION THREAD

**Discussion by USAMAWIZARD** [ant-media/Ant-Media-Server#5335] [FAQs] [answered]
How to disable one tracks from receiver client when there are multiple audio tracks

**Comment by yashtandon113** [comment, marked as answer]
### Method 1: Stop receiving track from the server side

You can use the `enableTrack` method provided by the WebRTC Adaptor to disable a particular track. This will prevent the server from sending packets for that specific track.


`webRTCAdaptor.enableTrack(mainTrackId, trackId, enabled);` 


-   `mainTrackId`: Unique ID for the main stream.
-   `trackId`: Unique ID for the track that you want to enable/disable data flow for.
-   `enabled`: Set to `true` or `false` to enable or disable the data flow.

### Method 2: Disable track on the client side

If you have a reference to the video element that is playing the video with multiple tracks, you can mute the first track using the following code:

```
const videoElement = document.getElementById('your-video-element-id');
videoElement.srcObject.getAudioTracks()[0].enabled = false;
```


This will disable the audio track of the video element.

**Reply by mekya** [reply]
I think there is also way to stop audio track on the server side. So that server does not send audio packets to the client. There should be a method like `enableTrack` in the SDK.   Is it possible to mention that as well?

**Reply by USAMAWIZARD** [reply]
Thanks for the feedback I have updated the answer
